src/assemblyItemsBrowser.js

```diff
@@ -192,13 +198,58 @@ module.exports = React.createClass({
 		}
 		
 		var currentAddress = this.state.currentAddress
-		if (props.vmTrace[vmTraceIndex].address) // there's a new context
-		{
-			var stack = this.state.callStack[vmTraceIndex]
-			currentAddress = stack[stack.length - 1]
-			this.resolveAddress(currentAddress, props)
-			Object.assign(stateChanges, { "currentAddress": currentAddress })
-		}
+        var addressIndex = this.shouldUpdateStateProperty("address", vmTraceIndex, previousIndex, props.vmTrace)
+        if (addressIndex > -1)
+        {
+            if (this.state.currentSelected === -1) // beginning of the trace
+            {
+                currentAddress = props.vmTrace[addressIndex].address
+                if (props.transaction.to === null) // contract creation
+                {
+                    this.cacheCode(currentAddress, props.transaction.input)
```

Comment:
This is not really true: The code is not stored there. We should analyze the vm trace, discover stack frames and associate code with them, not with addresses. But that can also be done in another task.